apps/worker/src/interfaces/publisher-job.interface.ts

```diff
@@ -0,0 +1,10 @@
+export interface IPFSJobData {
+  cid: string;
+  payloadLength: number;
+}
+
+export interface IPublisherJob {
+  id: string;
+  schemaId: number;
+  data: IPFSJobData;
+}
```

Comment:
publishQueue will hold this data format, we can change it as per growing requirement